src/test/StringCalculatorTest.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StringCalculatorTest {
```

Comment:
String is everywhere in test and in class name too.

Reply:
Done